src/main/java/com/minecolonies/coremod/entity/ai/minimal/EntityAIGoHome.java

```diff
@@ -155,10 +155,17 @@ private void handleSaturation(@NotNull final BlockPos pos)
                         }
                         else
                         {
-                            citizen.getInventoryCitizen().setInventorySlotContents(slotToSet, new ItemStack(stack.getItem(), 1));
+                            final ItemStack copy = stack.copy();
```

Comment:
I would rather that we don't mix feature/ bug fix together, make back porting less trivial